src/components/transaction/DataFormat.vue

```diff
@@ -101,34 +115,41 @@ export default defineComponent({
             isOverflowing,
             showOverflow,
             toggleOverflow,
-            maxHeight,
-            switchHeight,
-            useColor,
+            maxHeightStyle,
         };
     },
 });
 </script>
 
 <template>
-
-<div class="relative-position" :class="showOverflow ? '' : 'overflow-hidden'" :style=" (showOverflow || !isOverflowing) ? '' : `max-height: calc(${maxHeight}px - ${switchHeight}px)`">
-  <div class="row" v-if="actionName === 'transfer'" ref="dataBox">
-    <div class="col-12"><span class="text-bold">
-        <AccountFormat :account="transferData.from" type="account"></AccountFormat></span><span class="text-bold">&nbsp; → &nbsp;
-        <AccountFormat :account="transferData.to" type="account">&nbsp;</AccountFormat></span><span class="text-bold">{{ ' ' + transferData.quantity}}</span></div>
-    <div class="col-12">
-      <div class="text-weight-bold">memo:&nbsp;<span class="text-weight-regular" v-if="transferData.memo">{{transferData.memo}}</span><span class="text-weight-regular" v-else>n/a</span></div>
+<div
+    class="relative-position"
+    :class="{'overflow-hidden': !showOverflow, 'div-compressed': !showOverflow}"
```

Comment:
double-check this but I believe the quasar class `overflow-hidden` is redundant now due to `div-compressed` (doesn't appear to do anything when disabled in dev tools)
```suggestion
    :class="{ 'div-compressed': !showOverflow }"
```